engine/src/test/java/org/hawkular/alerts/engine/impl/ispn/IspnActionsServiceImplTest.java

```diff
@@ -34,8 +31,6 @@
     @BeforeClass
     public static void init() {
         try {
-            SmallRyeConfig config = ConfigUtils.configBuilder(true).build();
-            QuarkusConfigFactory.setConfig(config);
```

Comment:
I have no idea why this was added in the first place, I've never seen that before.

The ConfigUtils API we used no longer exists and tests work fine without this code, so I think it can be safely removed.

Reply:
We could, but I didn't have time to test 2.3.0 in any project until now so I'm sticking with 2.2.3 for now 😃

Reply:
Okay :) I still prefer just putting it on stage and see what happens. When it fails, we know we are missing tests :D